### PR TITLE
Delete realm file if schemas are incompatible

### LINKED ISSUE #3862: Add message asking to reinstall application instead of error when updating
### User Story

As a user I want to be notified about unavailability to upgrade the application with understandable message 

### Description

*Type*: Feature

*Summary*: It's not possible anymore to upgrade previous release version to the new one. An error appears when opening an upgraded app. We need a friendly informative message asking a user to reinstall the app

#### Expected behavior
Pop up with asking to uninstall and install the app appears

#### Actual behavior
Error `com.facebook.react.commonException...` pop up appears
aa

### Reproduction

- Install Status app release 0.9.16 version (from Play Store or from the [link](http://artifacts.status.im:8081/artifactory/nightlies-local/im.status.ethereum-28ec8a.apk))
- Upgrade the application to the latest develop build ([link](http://artifacts.status.im:8081/artifactory/nightlies-local/im.status.ethereum-714873.apk))

### Solution
[comment]: # (Please summarise the solution and provide a task list on what needs to be fixed.)
*Summary*: 

- Notify a user that the app couldn't be upgraded and should be uninstalled first in the Play Store description
- Create information message instead of error pop up to be appeared in case if a user has updated the app already

### Additional Information
* Status version: [apk](http://artifacts.status.im:8081/artifactory/nightlies-local/im.status.ethereum-714873.apk)
* Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by yenda**
fix #3862 

The issue is about showing a popup asking the user to reinstall, this PR propose instead to delete the realm file so that no user action is required. I tried to show a popup about that but the way the code is organized this would require more changes because a popup at that point would be immediatly hidden by the popup about rooted devices. If there is a clear enough warning in the update informations I think it is ok to just do it like this.

status: ready

### CODE REVIEW

**Review by janherich — APPROVED** [review]

File: src/status_im/translations/en.cljs [hunk 1]
```diff
@@ -32,6 +32,8 @@
    :photos-access-error                  "To grant the required photos permission, please go to your system settings and make sure that Status > Photos is selected."
 
    :status-console                       "Status Console"
+   :upgrade-status-error                 "It appears your phone was running a previous incompatible version of Status, we need to delete it to upgrade"
+   :delete                               "Delete"
```
Comment (flexsurfer) [inline src/status_im/translations/en.cljs]: we have delete at 267 line

**Comment by jeluard** [comment]
@yenda Is it WIP?

File: src/status_im/translations/en.cljs [hunk 2]
```diff
@@ -32,6 +32,7 @@
    :photos-access-error                  "To grant the required photos permission, please go to your system settings and make sure that Status > Photos is selected."
 
    :status-console                       "Status Console"
+   :delete                               "Delete"
```
Comment (flexsurfer) [inline src/status_im/translations/en.cljs]: can you revert this file?

**Comment by flexsurfer** [comment]
popup about rooted devices? whats this?

**Comment by yenda** [comment]
@flexsurfer when your device is rooted you get a popup warning you about security, search for root in the Android native code it's not in the react code if you want to see how it's done.

**Comment by yenda** [comment]
@flexsurfer I reverted the translation file

**Review by flexsurfer — APPROVED** [review]

**Pull request merged**